source/agora/node/Ledger.d

```diff
@@ -25,17 +26,18 @@ import agora.node.API;
 import vibe.core.log;
 
 import std.algorithm;
+import std.file;
+import std.path;
 
 /// Ditto
 public class Ledger
 {
-    /// data storage for all the blocks,
+    /// Blockstorage for all the blocks,
```

Comment:
```suggestion
    /// Blockstorage for all the blocks
```

And remove the next line which says `/// currently a single contiguous region to`, because the comment is outdated (and its cut-off, because the `/// improve locality of reference` part was removed).

Reply:
Fixed